ci/tsqa/tests/test_forward_proxy.py

```diff
@@ -0,0 +1,123 @@
+'''
```

Comment:
for the test engine what I have at the moment is here:
https://bitbucket.org/dragon512/reusable-gold-testing-system/wiki/Home 
For stuff extended for the traffic server is here:
https://github.com/apache/trafficserver/blob/master/tests/getting_started.md

I would suggest look at some of the test to get a basic feel of what you test probably will look like.